classes/Media.php

```diff
@@ -624,12 +624,9 @@ public static function ieCssSplitter($compiled_css, $cache_path, $refresh = fals
         $protocol_link = Tools::getCurrentUrlProtocolPrefix();
         //return cached css
         if (!$refresh) {
-            $cached_files = scandir($cache_path);
-            foreach ($cached_files as $file) {
-                if ($file != '.' && $file != '..') {
-                    $css_url = str_replace(_PS_ROOT_DIR_, '', $protocol_link.Tools::getMediaServer('').$cache_path.DIRECTORY_SEPARATOR.$file);
-                    $splitted_css[$css_url] = 'all';
-                }
+            foreach (array_diff(scandir($cache_path), array('..', '.')) as $file) {
```

Comment:
The issue with computing the `array_diff` within the `foreach` is that you make the code computing the same value in every loop. By taking the array_diff computation out of the loop and into a variable, you gain some valuable microseconds, I'm sure -- moreover because you computation performs a `scandir`, which touches upon the filesystem and therefore waaaaaay slower than any regular array creation :)